templates/experiments/experiment_session_view.html

```diff
@@ -40,6 +40,20 @@ <h3 class="text-base font-semibold leading-7 mt-8 pl-4">Experiment details</h3>
     <div class="max-w-5xl mt-4 border rounded-2xl">
       {% include "experiments/components/experiment_details.html" %}
     </div>
+    {% if experiment_session.get_participant_data %}
+      <div x-data="{showParticipantData: false}">
+        <h3 class="text-base font-semibold leading-7 mt-8 pl-4">
+          <div class="hover:cursor-pointer" @click="showParticipantData = !showParticipantData;">
+            <i x-show="!showParticipantData" class="fa fa-arrow-right" aria-hidden="true"></i>
+            <i x-cloak x-show="showParticipantData" class="fa fa-arrow-down" aria-hidden="true"></i>
+            Participant Data
+          </div>
+        </h3>
+        <div x-cloak x-show="showParticipantData" class="max-w-5xl mt-4 rounded-2xl">
+          {{ experiment_session.get_participant_data }}
```

Comment:
Suggestion:

Make this a tab UI with 2 tabs:

1. Key value pair list view
  This is the 'nice' view that shows each top level key in the data with it's associated value in a tabular style.
2. Raw view
  Show the formatted JSON data in a `<pre>` section.

Reply:
Formatted:

![formatted](https://github.com/dimagi/open-chat-studio/assets/64970009/ec7c544a-391d-46d0-a7d8-55ba6a076387)

Raw:

![image](https://github.com/dimagi/open-chat-studio/assets/64970009/432ad31c-f95c-46b1-89ff-3aaa08f98821)

Reply:
nice!